src/universal/common/components/AppHeader/AppHeader.tsx

```diff
@@ -9,7 +9,7 @@ export default class AppHeader extends React.Component<any, any> {
     return (
       <div className={css.AppHeader}>
         <div className={css.LogoContainer}>
-          <a href="/"><AppLogo /></a>
+          <a href="https://www.everreal.de"><AppLogo /></a>
```

Comment:
This has to point to the main website, `/` will point to `https://auth.everreal.co`, but should point to `https://www.everreal.co`.